docs/architecture-and-technical-specifications.md

```diff
@@ -434,31 +468,40 @@ attestation to access age-restricted content or services.
 
 *Service requests Proof of Age attestation (step 5\)*
 
-* The RP replies by requesting a Proof of Age, typically through one of the 
-following mechanisms:
+* The RP replies by requesting a Proof of Age, typically by providing all of 
+the following mechanisms:
 
-  * Same-Device App Integration: The User's browser or a native app opens the 
-  AVI directly (if on the same device).
+  * A file containing the minimum age required, and a nonce for the request.
 
-  * Cross-Device QR Code: The RP displays a QR code that the User scans with 
-  their mobile device to open the AVI.
+  * A text based token containing the same information, to use via copy & 
+paste or similar functions, using e.g. Base64 encoded JSON, or Base45 encoded
+CBOR as in the EU COVID passports.
 
-*Confirmation and presentation (step 5\)*
+  * A QR code that can be scanned via the AVI to read the information. 
 
+*Confirmation and presentation (step 6\)*
+
+* The User opens the AVI and uploads the file, pastes the token, or scans the 
+QR code to initiate the request flow.
+  
 * The AVI receives the Proof of Age request and presents it to the User. The 
 User reviews the request details, verifies the information, and confirms the 
-transaction to proceed.
+transaction to proceed. The process SHALL be completely offline and not 
+transmit any information to any server.
```

Comment:
Given this proposal this sounds like it may be sufficient for it, but it should be explicitly stated in the requirements that any external verification service, whether public or private, MUST NOT have any ability to track how many times a particular person attempts to verify their age across any/all services, including if they request it at all.